source/web-documentroot.rst

```diff
@@ -26,6 +26,30 @@ You can create folders (and symlinks) in the form of ``/var/www/virtual/<usernam
 
 .. warning:: Do not delete ``/var/www/virtual/<username>/html``. If this folder doesn't exist, the RewriteRules implementing the additional DocumentRoots don't work, so all your domains will be inaccessible.
 
+Example how to change DocumentRoot location
+-------------------------------------------
+
+Many PHP apps like Symfony provide their own public webfolder within their folder structure. It is recommended to only make this folder accessible by the webserver. This can be achieved, for example, with the following procedure:
+
+.. code-block:: console
+
+  # in /var/www/virtual/$USER
+  [eliza@dolittle eliza]$ rm -f html/nocontent.html; rmdir html
```

Comment:
one command per line please, keep it simple.

Reply:
thats copy& paste from Lab https://github.com/Uberspace/lab/commit/723dbafd3af9c63fd912621255b317b8de478cf2#diff-cfa113bbc39ea5346b4a3098408bd54123add9f21d41148c9e23d06bf25fca2a :stuck_out_tongue_winking_eye: